packages/widgets/src/blocks/legacy-widget/edit/index.js

```diff
@@ -94,12 +94,7 @@ function NotEmpty( {
 } ) {
 	const [ hasPreview, setHasPreview ] = useState( null );
 
-	const {
-		widgetType,
-		hasResolvedWidgetType,
-		isWidgetTypeHidden,
-		isNavigationMode,
-	} = useSelect(
+	const { widgetType, hasResolvedWidgetType, isNavigationMode } = useSelect(
```

Comment:
Please remove `isWidgetTypeHidden` from the returned object too. Unfortunately the linter doesn't detect this.